fe/fe-core/src/main/java/org/apache/doris/planner/DataPartition.java

```diff
@@ -53,18 +54,24 @@ public class DataPartition {
     public static final DataPartition RANDOM = new DataPartition(TPartitionType.RANDOM);
 
     private final TPartitionType type;
+    private final THashType hashType;
 
     // for hash partition: exprs used to compute hash value
     private ImmutableList<Expr> partitionExprs = ImmutableList.of();
 
     public DataPartition(TPartitionType type, List<Expr> exprs) {
+        this(type, exprs, THashType.XXHASH64);
```

Comment:
Previously, the default hash type is CRC32, I think we should not change it

Reply:
fixed